### ISSUE TITLE
[suggestion] Farming mode/options 

### ISSUE THREAD

**Issue by netlore** [Wuild/GatherLite#40]
Would be great to have an option to only show certain nodes... if I'm in an area farming for 1 or 2 particular types of items.

**Comment by GrandMasterLynx** [comment]
That would be good to have

**Comment by Wuild** [comment]
Do you mean like a filter? only show for example "Iron Veins" ??

**Comment by pquerner** [comment]
Yes thats what I understand is asked. It would be a great addition.

**Comment by netlore** [comment]
Yes, exactly as you say... "_only show for example "Iron Veins"_"

**Comment by jshipley** [comment]
I'd love to be able to hide fish. It would be one thing if it only showed fishing nodes, but I don't really want to see icons for every place I've ever tried fishing.

**Comment by Wuild** [comment]
> I'd love to be able to hide fish. It would be one thing if it only showed fishing nodes, but I don't really want to see icons for every place I've ever tried fishing.

By clicking the minimap button you can select what nodes to display.

**Comment by jshipley** [comment]
Thanks, I was looking all over the settings page for that. I didn't see it, because the first thing I usually do for any addon is get rid of the minimap button if I can.

**Comment by Wuild** [comment]
tracking is now available on the settings page aswell.
